File: PHP/Questões uri/1037.php
<?php

/**
 * Escreva a sua solução aqui
 * Code your solution here
 * Escriba su solución aquí
 */
$t=trim(fgets(STDIN));
if ($t < 0 or $t>100) {
  echo "Fora de intervalo"."\n";
} 
elseif ($t>=0 and $t<=25){
  echo "Intervalo [0,25]"."\n";
}
elseif ($t>=25.01 and $t<=50){
  echo "Intervalo (25,50]"."\n";
}
elseif ($t>=50.01 and $t<=75){
  echo "Intervalo (50,75]"."\n";
}
elseif ($t>=75.01 and $t<=100){
  echo "Intervalo (75,100]"."\n";
}


?>

File: ESTUDO DE LINGUAGENS/PYTHON/Questões Uri/1037.py
valor=(float(input()))

if valor >= 0 and valor <= 25:
  print("Intervalo [0,25]")

if valor>= 25.01 and valor <= 50:
  print("Intervalo (25,50]")

if valor>= 50.01 and valor <=75:
  print("Intervalo (50,75]")

if valor>= 75.01 and valor <=100:
  print("Intervalo (75,100]")

if valor > 100 or valor < 0:
  print("Fora de intervalo")
